src/components/nav/nav.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
```

Comment:
Because body styles are unique and apply for the whole application it's better to move these styles to src/styles/index.scss